os/customizing-docker.md

```diff
@@ -309,17 +309,14 @@ The easiest way to use these new ulimits on all of your machines is via cloud-co
 ```yaml
 #cloud-config
 
-write_files:
-    - path: /etc/systemd/system/docker.service.d/increase-ulimit.conf
-      owner: core:core
-      permissions: 0644
```

Comment:
@robszumski I *think* that the units will get default and appropriate permissions when created in this fashion (specified in terms of their parent unit). Before this change, we needed to specify the file mode because we were creating it with a brute force `write_files:`. Now we don't. @kayrus can you confirm if I'm correct?